skins/laika/src/components/layout/Footer.vue

```diff
@@ -12,40 +12,33 @@
 							</div>
 						</div>
 						<div class="level-item">
-							<div class="bank-data-content">
-								<p><strong>Kontoinhaber:</strong> Wikimedia Foerdergesellschaft</p>
-								<p>
-									<strong>{{ $t( 'iban')  }}:</strong>
-									<span
-											class="spacer"
-											v-for="( ibanPartial, index ) in formatIban()"
-											:key="index">{{ ibanPartial }}</span>
-								</p>
-								<p><strong>{{ $t( 'bic')  }}:</strong> {{ $t( 'operator_account_bic')  }}</p>
-								<p>Bank für Sozialwirtschaft</p>
-								<p><strong>{{ $t( 'account_number')  }}:</strong> {{ $t( 'operator_account_number')  }}</p>
-								<p><strong>{{ $t( 'blz')  }}:</strong> {{ $t( 'operator_account_blz')  }}</p>
-							</div>
+							<BankData></BankData>
 						</div>
 					</div>
 				</div>
-				<div class="column is-one-third is-half-desktop">
-					<ul class="list-menu">
-						<li v-for="( link, index ) in footerMenu" :key="index">
-							<a :href="link.url" :key="link.id">
-								<span>{{ $t( 'footer_menu_' + link.id ) }}</span>
-							</a>
-						</li>
-					</ul>
+				<div class="column is-one-third">
+					<div class="level">
+						<div class="level-item">
+							<ul class="list-menu">
+								<li v-for="( link, index ) in footerMenu" :key="index">
+									<a :href="link.url" :key="link.id">
+										<span>{{ $t( 'footer_menu_' + link.id ) }}</span>
+									</a>
```

Comment:
this indentation is killing me :sweat: 
is the `<span>` needed in the link?

Reply:
Now that I look at it again, it is pretty crazy, yeah. The span can indeed be removed though which I have done now.